MYshort.bib

```diff
@@ -316,6 +316,7 @@ @String{SEN
 @String{SIGPLAN = "SIGPLAN Not."}
 @String{SME = "SME"}
 @String{JSME = "JSME"}
+@String{JSMRP = "J.\ Softw.\ Maint: Res.\ Pract."}
```

Comment:
Noticeably longer than the others here. Is this intended?

Reply:
This journal does not seem to have an official abbreviation. It seems as if journals love to abbreviate themselves in such awful ways. Do we have a policy for this @tthuem ?

Reply:
Decision is up to me.

Reply:
I chose JSMRP now but documented that this is not the official documentation but "J.\ Softw.\ Maint: Res.\ Pract.".